my-webxr-app/src/App.tsx

```diff
@@ -90,6 +53,27 @@ export default function App() {
     };
     initializeDB();
   }, [dbName, storeName]);
+
+  // TEST PlottedData
+  const exampleDataPoints = [[-1, -1, -1], [2, 3, 0], [4, 3, 0], [1, 1, 1], [3, 2, 2]];
+  const dataPoints = exampleDataPoints.map((point) => new DataPoint(point[0], point[1], point[2]));
+
+  const [plottedDataPoints, setPlottedDataPoints] = useState<JSX.Element[]>([]);
+  useEffect(() => {
+    if (dataPoints.length > 0) {
+      const plottedPoints = createGraphingDataPoints(
+        dataPoints,
+        'columnX',
```

Comment:
it might be redundant and space-consuming to store the 3 headers within each point though we can improve this in a follow-up PR